pkg/filesystem/local_directory_windows.go

```diff
@@ -584,44 +585,50 @@ func (d *localDirectory) Remove(name path.Component) error {
 	return err
 }
 
-// On NTFS mount point is a reparse point, no need to unmount.
 func (d *localDirectory) RemoveAllChildren() error {
+	return d.RemoveAllChildren(nil)
+}
+
+// On NTFS mount point is a reparse point, no need to unmount.
+func (d *localDirectory) RemoveAllChildren(dPath *path.Trace) error {
```

Comment:
Does this even compile? It has the same name as the function above. Please make it lowercase, so that it's private.

Reply:
Yeah, no. I forgot to go to the windows computer.

Reply:
Done, in subsequent work.